frameworks/person-escort-record/questions/history-of-self-harm-recency.yml

```diff
@@ -1,24 +1,10 @@
-type: radio
-question: How recently did self-harm occur?
-description: Recency of self-harm
+type: text
+question: Date when the last incident took place
+hint: For example 28/10/2019 or 28 October 2019
```

Comment:
this was marked as tbd in ticket, let me know your thoughts

Reply:
Looks fine to me (if both date formats are acceptble).